airbyte-integrations/bases/debezium/src/main/java/io/airbyte/integrations/debezium/internals/mongodb/MongoDbDebeziumStateUtil.java

```diff
@@ -79,7 +81,7 @@ public static JsonNode formatState(final String database, final String replicaSe
     final List<Object> key = generateOffsetKey(database, replicaSet);
 
     final Map<String, Object> value = new HashMap<>();
-    value.put(MongoDbDebeziumConstants.OffsetState.VALUE_SECONDS, timestamp.getTime());
+    value.put(VALUE_SECONDS, timestamp.getTime());
```

Comment:
Nit:  I'm not sure what is going on with Git and IntelliJ, but this keeps yo-yo'ing between using the fully qualified name and a static import.  All of the other constants referenced from that class in this class are using the fully-qualified name, if you want to undo/put this one back so they are consistent.

Reply:
I've fixed this a few times already, but it seems that every time I merge the base branch into this branch it undoes it 😞